### PR TITLE
[DCWL-1835] Detach customs-api-common 

### PR DESCRIPTION
**Pull request by JakeReid2020**
Please ensure we are following our Ways of Working for Pull Requests: https://confluence.tools.tax.service.gov.uk/display/BTL/Ways+of+Working+for+Pull+Requests

### CODE REVIEW

**Comment by kevin-co-hmrc** [comment]
Can we sort out the title of the PR? It's no longer work in progress.

File: app/uk/gov/hmrc/customs/inventorylinking/imports/CustomsApiCommonModule.scala [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ */
+
+package uk.gov.hmrc.customs.inventorylinking.imports
+
+import play.api.inject.{Binding, Module}
+import play.api.{Configuration, Environment}
+import uk.gov.hmrc.http.HttpClient
+import uk.gov.hmrc.play.bootstrap.http.DefaultHttpClient
+
+class CustomsApiCommonModule extends Module {
```
Comment (kevin-co-hmrc) [inline app/uk/gov/hmrc/customs/inventorylinking/imports/CustomsApiCommonModule.scala:24]: Could we rename the module to something more relevant?

File: test/resources/public/api/conf/1.0/application.raml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+documentation:
+  - title: Overview
+    content: !include docs/overview.md
+  - title: Versioning
+    content: !include https://api-documentation/assets/common/docs/versioning.md
+  - title: Errors
+    content: !include https://api-documentation/assets/common/docs/errors.md
+
+/customs-wco:
```
Comment (kevin-co-hmrc) [inline test/resources/public/api/conf/1.0/application.raml:24]: Is this relevant?

Comment (JakeReid2020) [reply test/resources/public/api/conf/1.0/application.raml:24]: It's used for the Documentation Controller Spec

Comment (kevin-co-hmrc) [reply test/resources/public/api/conf/1.0/application.raml:24]: Ah yes I see. The test data had `customs-wco` which is what confused me, but it seems like the tests are simply asserting that the data is served, irrespective of the actual content.

**Review by kevin-co-hmrc — APPROVED** [review]
Looks good 👍🏼

**Pull request merged**